dashboard/test/controllers/scripts_controller_test.rb

```diff
@@ -1734,7 +1744,7 @@ class CourseInDevelopmentTests < ActionController::TestCase
     Unit.expects(:get_without_cache).with(@migrated_unit.name, with_associated_models: true).returns(@migrated_unit).once
     get :edit, params: {id: @migrated_unit.name}
 
-    Unit.expects(:get_from_cache).with(@migrated_unit.name, raise_exceptions: false).returns(@migrated_unit).once
+    Unit.expects(:get_from_cache).with(@migrated_unit.name, raise_exceptions: false).returns(@migrated_unit).twice
```

Comment:
It looks like you found a fragile test. how about making it a bit less fragile:
```suggestion
    Unit.expects(:get_from_cache).with(@migrated_unit.name, raise_exceptions: false).returns(@migrated_unit).at_least_once
```